org.eclipse.acute/src/org/eclipse/acute/dotnetrun/DotnetRunDelegate.java

```diff
@@ -93,11 +98,14 @@ public void launch(IEditorPart editor, String mode) {
 	@Override
 	public void launch(ILaunchConfiguration configuration, String mode, ILaunch launch, IProgressMonitor monitor)
 			throws CoreException {
-		String projectLocation = configuration.getAttribute(DebugPlugin.ATTR_WORKING_DIRECTORY, "");
+		File projectFileLocation = new File(configuration.getAttribute(DebugPlugin.ATTR_WORKING_DIRECTORY, ""));
```

Comment:
The ATTR_WORKING_DIRECTORY here has a specific meaning, which shouldn't be overriden. If you need the path to the project description file, you should create your own attrbute instead to avoid collisions in usage.